en/docs/learn/configuring-the-realm.md

```diff
@@ -16,12 +17,8 @@ The following diagram illustrates the required configurations and
 repositories:  
 ![configuring-the-realm.png](../assets/img/using-wso2-identity-server/configuring-the-realm.png)
 
-The following sections include instructions on the above required
-configurations and repositories:
+For more information on Configuring master-datasources.xml, click
+[here](../../administer/configuring-master-datasources.xml).
```

Comment:
shouldn't we change this now according to the toml config model?

Reply:
Need to check if we need these pages in our administer docs.

Reply:
removed now